### PR TITLE
Add ali3ncat's file

### PR DESCRIPTION
**Pull request by c4thu1lhu**
create branch
create file
make a commit
open a pull request

### CODE REVIEW

File: _posts/0000-01-02-ali3ncat.md/second_slide [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (ali3ncat) [inline _posts/0000-01-02-ali3ncat.md/second_slide:5]: Los pedos pesan?

Comment (ali3ncat) [inline _posts/0000-01-02-ali3ncat.md/second_slide:5]: This wound sound strange to ask, but
this

**Review by c4thu1lhu — COMMENTED** [review]
The line is "Do you know why our eyebrows are similar?" and he responded "Wait what?"

**Comment by ali3ncat** [comment]
Sorry, that was a weird thing to ask

**Pull request merged**